projects/plugins/protect/src/js/components/summary/styles.module.scss

```diff
@@ -17,24 +17,25 @@
 
 .summary__icon {
 	margin-left: -4px; // this fix a blank space on right/left from @wordpress/icons
-	margin-right: var(--spacing-base);
+	margin-right: var( --spacing-base ); // 8px
 }
 
 .summary__notice {
-	margin-top: 24px;
+	margin-top: calc( var( --spacing-base ) * 3 ); // 24px
 
 	@media ( min-width: 960px ) {
 		margin-top: 0;
 	}
 }
 
 .summary__scan-button {
-	margin-top: 16px;
+	margin-top: calc( var( --spacing-base ) * 2 ); // 16px
 	width: 100%;
 
 	@media ( min-width: 960px ) {
 		margin-top: 0;
-		margin-left: 16px;
+		margin-left: 
```

Comment:
Looks like we've got an extra `margin-left` declaration here that can be removed. I _think_ that should fix the current `plugins/protect` build issues as well.

Reply:
Thanks for catching this, resolved via [273abcb](https://github.com/Automattic/jetpack/pull/27175/commits/273abcb2ec64612e3370a79c347047c90ac4221e)!